AppSettingsManagerAPI/AppSettingsManagerApi.Infrastructure/Migrations/20230227202735_Permissions.Designer.cs

```diff
@@ -0,0 +1,174 @@
+﻿// <auto-generated />
+using System;
+using AppSettingsManagerApi.Infrastructure.MySql;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+
+#nullable disable
+
+namespace AppSettingsManagerApi.Infrastructure.Migrations
+{
+    [DbContext(typeof(SettingsContext))]
+    [Migration("20230227202735_Permissions")]
+    partial class Permissions
+    {
+        protected override void BuildTargetModel(ModelBuilder modelBuilder)
+        {
+#pragma warning disable 612, 618
+            modelBuilder
+                .HasAnnotation("ProductVersion", "6.0.10")
+                .HasAnnotation("Relational:MaxIdentifierLength", 64);
+
+            modelBuilder.Entity("AppSettingsManagerApi.Infrastructure.MySql.Permission", b =>
+                {
+                    b.Property<string>("UserId")
+                        .HasColumnType("varchar(36)");
+
+                    b.Property<string>("SettingGroupId")
+                        .HasColumnType("varchar(255)");
+
+                    b.Property<string>("ApprovedBy")
+                        .IsRequired()
+                        .HasColumnType("longtext");
+
+                    b.Property<int>("CurrentPermissionLevel")
+                        .HasColumnType("int");
+
+                    b.Property<int>("RequestedPermissionLevel")
+                        .HasColumnType("int");
+
+                    b.Property<bool>("WaitingForApproval")
+                        .HasColumnType("tinyint(1)");
+
+                    b.HasKey("UserId", "SettingGroupId");
+
+                    b.HasIndex("SettingGroupId");
+
+                    b.ToTable("Permissions");
+                });
```

Comment:
this all seems pretty straightforward so far, don't see anything out of order nor do i have any questions

Reply:
These files, while they do need to be included in the repository, you can basically ignore. They're auto-generated by entity framework whenever a new "migration" is added